src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.html

```diff
@@ -11,14 +11,7 @@
         <span>{{ collection.name }}</span>
         <div class="collection__image" [style.background-image]="'url(assets/images/collections/' + learningObject?.collection + '.png)'"></div>
       </div>
-      <div *ngIf="learningObject?.status" class="top__status" [ngClass]="learningObject?.status" tip="{{ states?.get(learningObject.status)?.tip }}"
-        tipDisabled="{{ !states?.get(learningObject.status)?.tip }}" tipLocation="bottom">
-        <span *ngIf="learningObject?.status === 'unreleased'"><i class="fas fa-eye-slash"></i></span>
-        <span *ngIf="learningObject?.status === 'waiting'"><i class="fas fa-hourglass"></i></span>
-        <span *ngIf="learningObject?.status === 'review'"><i class="fas fa-sync"></i></span>
-        <span *ngIf="learningObject?.status === 'released'"><i class="fas fa-eye"></i></span>
-        <span *ngIf="learningObject?.status === 'denied'"><i class="fas fa-ban"></i></span>
-      </div>
+      <clark-lo-status-indicator [learningObject]="learningObject" [states]="states"></clark-lo-status-indicator>
```

Comment:
I'd appreciate some feedback on the name of this selector. I feel like `clark-learning-object-status-indicator` is a bit verbose, and `clark-status-indicator` is too nondescript. I abbreviated learning object to "lo" as done elsewhere in the app, but I don't know that I'm happy with that. Thoughts?

Reply:
I like `clark-lo`. More descriptive than `clark-status`, but no so verbose. This was a good in between :+1: